openapi/nexus.json

```diff
@@ -20438,7 +20439,7 @@
         ]
       },
       "VpcFirewallRuleFilter": {
-        "description": "Filter for a firewall rule. A given packet must match every field that is present for the rule to apply to it. A packet matches a field if any entry in that field matches the packet.",
+        "description": "Filters reduce the scope of a firewall rule. Without filters, the rule applies to all packets to the targets (or from the targets, if it's an outbound rule). With multiple filters, the rule applies only to packets matching ALL filters. The maximum number of each type of filter is 256.",
```

Comment:
@karencfv I brought in something more like the console explanation here. I think it's an improvement, the only issue is that it's hard to find on the docs site, buried in the request body.

<img width="1249" alt="image" src="https://github.com/user-attachments/assets/792f2521-a570-42e2-8437-412847b96757">

What might be nicer would be to put some basic summary of all of it here at the top... OR we could just link to the guide, which is pretty nice: https://docs.oxide.computer/guides/configuring-guest-networking#_firewall_rules

<img width="1231" alt="image" src="https://github.com/user-attachments/assets/848f3d17-d3d6-48d2-9ad9-aa507df98b45">

Reply:
Oh nice! This is great. I'd like to keep this here so the SDKs inherit the descriptions, but it would be really great to add the text from the second image as well for the docs. Would it be too weird to have it all?

Reply:
Yeah, I’ll add more. I don’t think anyone will be mad at us for having too good of documentation.